benchmarks/istanbul.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
```

Comment:
Please follow the project's code formatting. Run `npm run lint` locally.

Reply:
okey, I'm fixed errors